extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -108,6 +112,21 @@ core::Property ConsumeWindowsEventLog::ResolveAsAttributes(
 	withDescription("If true, any metadata that is resolved ( such as IDs or keyword metadata ) will be placed into attributes, otherwise it will be replaced in the XML or text output")->
 	build());
 
+
+core::Property ConsumeWindowsEventLog::EventHeaderDelimiter(
+	core::PropertyBuilder::createProperty("Event Header Delimiter")->
```

Comment:
Guess we don't run linter for this module. This would surely break it.

Reply:
no linter in windows.